_posts/2021-06-08-moveit-vs-moveit2.md

```diff
@@ -0,0 +1,46 @@
+---
+author: Henning Kayser
+comments: false
+date: 2021-06-08
+layout: post
+title: MoveIt 2 Status Update and Feature Comparison
+media_type: image
+media_link: /assets/images/blog_posts/2021_06_08_comparison.png
+description: What new features are only available in MoveIt 2
+categories:
+- ROS2
+- MoveIt
+---
+
+With the release of ROS 2 Galactic, we would like to share the new features only available in MoveIt 2. We hope to encourage the community to upgrade to MoveIt 2 as soon as possible. MoveIt 2 is out of beta.
+
+
+|                                                                                                                                                    | MoveIt 1 | MoveIt 2       |
+| -------------------------------------------------------------------------------------------------------------------------------------------------- | -------- | -------------- |
+| Motion Planning                                                                                                                                    | ✓        | ✓              |
+| Inverse Kinematics                                                                                                                                 | ✓        | ✓              |
+| Perception                                                                                                                                         | ✓        | ✓              |
+| Grasping                                                                                                                                           | ✓        | ✓              |
+| Setup Assistant                                                                                                                                    | ✓        | In Development |
+| Game controller integration for Servo                                                                                                              | ✓        | ✓              |
```

Comment:
```suggestion
| Game Controller Integration for Servo                                                                                                              | ✓        | ✓              |
```
Need to make sure all of these are in Title Case

Reply:
Thanks @auman66 !